content/pt-br/docs/reference/issues-security/issues.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Rastreador de Issue Kubernetes
+weight: 10
+aliases: [/pt-br/cve/,/pt-br/cves/]
+---
+
+Para reportar um problema de segurança, siga [processo de divulgação de issues do Kubernetes](/docs/reference/issues-security/security/#report-a-vulnerability).
+
+O trabalho no código do Kubernetes e os problemas de segurança podem ser encontrados usando [ GitHub Issues ](https://github.com/kubernetes/kubernetes/issues/).
+
+* Oficial [lista de CVEs conhecidos](/docs/reference/issues-security/official-cve-feed/)
+  (vulnerabilidades de segurança) que foram anunciados pelo
+  [comitê de resposta de segurança](https://github.com/kubernetes/committee-security-response)
+* [Problemas do gitHub relacionados ao CVE](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Aarea%2Fsecurity+in%3Atitle+CVE)
```

Comment:
Issues are listed and not on GitHub. What do you think "Questões relacionadas ao CVE"